zbus/src/address.rs

```diff
@@ -165,6 +165,9 @@ pub enum Address {
     /// type of `stream` is `vsock::VsockStream` with `vsock` feature and `tokio_vsock::VsockStream`
     /// with `tokio-vsock` feature.
     Vsock(VsockAddress),
+    /// A listenable address using a specified path, in which a socket file with a random file name
```

Comment:
* "using a" -> "using the".
* can we document a bit better, like where does this come from (perhaps link to the appropriate part of the spec as well)?

Reply:
this is taken almost straight from dbus spec, adding a link